src/rest-server/docs/swagger.yaml

```diff
@@ -1,53 +1,45 @@
 openapi: 3.0.1
 info:
   title: OpenPAI RESTful API
-  description: 'Open Platform for AI RESTful API docs.'
+  description: Open Platform for AI RESTful API docs.
   license:
     name: MIT License
-    url: https://github.com/microsoft/pai/blob/master/LICENSE
+    url: "https://github.com/microsoft/pai/blob/master/LICENSE"
```

Comment:
we use single quotes `'` instead of `"` in `rest-server/` and `webportal/`, better to keep consistent

Reply:
this style is preferred by MS swagger, however, I would convert it for consistency

Reply:
Done